api.planx.uk/helpers.ts

```diff
@@ -247,6 +247,10 @@ const getFormattedEnvironment = (): string => {
     const pizzaNumber = new URL(process.env.API_URL_EXT!).href.split(".")[1];
     environment += ` ${pizzaNumber}`;
   }
+  // For readability redefine development as local for slack warnings
+  if (environment === "development") {
+    environment = "local";
```

Comment:
_Context:_

I didn't expect to see a Slack message when my local test of running the sanitisation failed. I think the message was very useful but my first (incorrect) instinct was that I'd triggered something in a live environment. 

I was relieved when I saw "Development" although with it being the end of the day and being tired and I did wonder if it was referring to staging and if I'd misunderstood the `.env` setup for the webhook.

Realistically I don't think I'll make this mistake again and I'm happy to leave this as "Development" if there are any strong opinions.